docs/install.md

```diff
@@ -109,6 +109,27 @@ You should never set any folder or file to permission level `777`, as this permi
 
 By default Flarum's directory structure includes a `public` directory which contains only publicly-accessible files. This is a security best-practice, ensuring that all sensitive source code files are completely inaccessible from the web root.
 
+::: vue
```

Comment:
I'm not sure this is the best place for this. Maybe under 'Installing' would be better? Also, this might be better in a separate PR.

Reply:
sonds good, I'll do that

Reply:
this change has been reverted